.circleci/config.yml

```diff
@@ -63,17 +63,21 @@ jobs:
       language:
         description: "Language"
         type: string
+      stack-tag:
+        description: "The tag of the run stack (ex. '20' for heroku/heroku:20)"
+        type: string
     docker:
       - image: heroku/pack-runner:latest
     steps:
       - run: git clone https://github.com/heroku/<< parameters.language >>-getting-started.git getting_started
       - setup_remote_docker
       - attach_workspace:
           at: /tmp/workspace
-      - run: docker load < /tmp/workspace/pack-18-build.tar
-      - run: docker load < /tmp/workspace/pack-18-run.tar
-      - run: docker load < /tmp/workspace/buildpacks-18.tar
-      - run: pack build pack-getting-started --path getting_started --builder heroku/buildpacks:18 --no-pull
+      - run: docker load < /tmp/workspace/pack-<< parameters.stack-tag >>-build.tar
+      - run: docker load < /tmp/workspace/pack-<< parameters.stack-tag >>-run.tar
+      - run: docker load < /tmp/workspace/buildpacks-<< parameters.stack-tag >>.tar
+      - run: docker pull buildpacksio/lifecycle:0.9.2
```

Comment:
I'm not sure why this was needed

Reply:
Because `heroku:buildpacks:20` is not trusted in `pack`. Therefore, we are falling back to `pack` running in untrusted mode and pulling the lifecycle image for the registry steps.

Reply:
I would rather see us trust the builder explicitly if we expect to be able to get it trusted in `pack`, rather than pulling specific lifecycle versions that will shift as `pack` gets updated.

Reply:
yea, maybe we should skip the smoke testing and publish here until then?

Reply:
I think I'd be ok with a `pack trust-builder heroku/buildpacks:20`. It should work fine regardless of trust/no-trust for users that find it and use it. It just makes CI annoying because of the specific image pull we have to do.

Reply:
fixed